overlays/intel-nuc.nix

```diff
@@ -0,0 +1,9 @@
+
+final: prev: {
+    spectrum-rootfs = prev.spectrum-rootfs.overrideAttrs (old: {
+        patches = [ ./patches/0001-Open-USB-serial-terminal-at-boot.patch ];
```

Comment:
We definitely must add comment to why this patch is applied. (I know). Even the patch does not comment *why* but only declares it opens the USB serial terminal. In any reference or user configuration this is considered a security risk, in development/debugging configuration this is valid but must be explicitly made clear why and what is the scope. FYI @tervis-unikie

Reply:
For the record - `ttyS0` is not acceptable, secure default in the upstream either.

Reply:
This probably is the subject for upstreaming to Spectrum (currently discussing this with Alyssa)